chart/templates/webserver/webserver-deployment.yaml

```diff
@@ -129,6 +129,12 @@ spec:
           resources:
 {{ toYaml .Values.webserver.resources | indent 12 }}
           volumeMounts:
+{{- if semverCompare ">=1.10.12" .Values.airflowVersion }}
```

Comment:
Should it be also <2.0.0 ?

Reply:
Also @jedcunningham @kaxil  -> I am not sure what will be the policy about supporting 1.10 in the chart (See https://apache-airflow.slack.com/archives/CCV3FV9KL/p1631564642172400 for context of that change)

Reply:
It probably should be present for 2.0.0+ too, as it is config, even if it isn't being used.

Reply:
Currently, there is not much maintenance effort for supporting 1.x so I think we should support it for now